storage_service/locations/api/resources.py

```diff
@@ -670,7 +671,12 @@ def pointer_file_request(self, request, bundle, **kwargs):
 
     @_custom_endpoint(expected_methods=['get'])
     def check_fixity_request(self, request, bundle, **kwargs):
-        success, failures, message = bundle.obj.check_fixity()
+        try:
+            success, failures, message = bundle.obj.check_fixity()
+        except:
```

Comment:
Only catch the exception(s) you're expecting here, or at a minimum `except Exception`.  Maybe a logging message with a traceback too?